cdmtaskservice/nersc/manager.py

```diff
@@ -0,0 +1,181 @@
+'''
```

Comment:
I assume the script is manually tested in NERSC? Do you think it's useful to do a readme doc with sample scripts so that it's easier for people to understand the process? Maybe not now but later?

Reply:
I'm going to suggest we revisit this comment in the next PR which will show how the scripts are used.